<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Surefire suite]" verbose="0">
  <test thread-count="5" name="Surefire test(failed)" verbose="0">
    <classes>
      <class name="codenbox.testCases.LoginDataDrivenTest">
        <methods>
          <include name="setUp"/>
          <include name="tearDown"/>
          <include name="verify_loginDDT"/>
        </methods>
      </class> <!-- codenbox.testCases.LoginDataDrivenTest -->
      <class name="codenbox.testCases.HomeTest">
        <methods>
          <include name="setUp"/>
          <include name="tearDown"/>
          <include name="homeAccount"/>
        </methods>
      </class> <!-- codenbox.testCases.HomeTest -->
      <class name="codenbox.testCases.LoginTest">
        <methods>
          <include name="setUp"/>
          <include name="tearDown"/>
          <include name="verify_login"/>
        </methods>
      </class> <!-- codenbox.testCases.LoginTest -->
      <class name="codenbox.testCases.RegistrationTest">
        <methods>
          <include name="setUp"/>
          <include name="tearDown"/>
          <include name="verify_refistration"/>
        </methods>
      </class> <!-- codenbox.testCases.RegistrationTest -->
    </classes>
  </test> <!-- Surefire test(failed) -->
</suite> <!-- Failed suite [Surefire suite] -->
